_episodes/00-Pre-requisites.md

```diff
@@ -20,59 +20,61 @@ keypoints:
 
 ## R and RStudio
 
-R is an open source programming language originally designed and implemented by statisticians primarily for statistical analysis. It includes high quality graphics capabilities and tools for basic data analysis and reading and writing data to/from files.
-Because R is open source and is supported by a large community of developers and users there is a very large selection of 3rd party add-on libraries available which are freely available and can easily be added to the core R system making even more analysis techniques readily available.  
-R is a scripted language. Rather than pointing and clicking in a GUI environment you actual write code statements to ask R to do something for you. This has the advantage of providing a permanent record of what was done and allows for peer review of the work undertaken.
+R is an open source programming language originally designed and implemented by statisticians for statistical analysis. It includes high quality graphics capabilities, tools for basic data analysis, and reading and writing data to/from files.
```

Comment:
here the "tools" refers to "tools for basic data analysis" and "tools for reading and writing data" so the comma is wrong in this case.